components/chart/src/selection-area/selection-area.ts

```diff
@@ -825,12 +848,30 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
 
     // handling of the overlay for the timestamp
     if (this._chart._timestamp) {
+      getElementRefStream<HTMLDivElement>(
+        this._chart._timestamp._stateChanges,
+        this._destroy$,
+        this._chart._timestamp._timestampElementRef,
+        this._zone,
+      )
+        .pipe(
+          throttleTime(0, animationFrameScheduler),
```

Comment:
Please remove throttling here is no need for it. This stream does not fire that much